### PR TITLE
add the processed ebook files.

### PR DESCRIPTION
**Pull request by BillWagner**
This is the third eBook that we are publishing on docs

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/standard/modern-web-apps-azure-architecture/?branch=pr-en-us-3357)

### CODE REVIEW

**Comment by BillWagner** [comment]
The build is passing, and this is now ready for review.

/cc @JRAlexander @CamSoper

File: docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+[Define your CD release process](https://www.visualstudio.com/en-us/docs/build/apps/aspnet/aspnetcore-to-azure#cd)
+
+Once your CI/CD pipeline is configured, you can simply make updates to your web app and commit them to source control to have them deployed.
+
+### Workflow for developing Azure-hosted ASP.NET Core applications
+
+Once you have configured your Azure account and your CI/CD process, developing Azure-hosted ASP.NET Core applications is simple. The following are the basic steps you usually take when building an ASP.NET Core app, hosted in Azure App Service as a Web App, as illustrated in Figure 10-3.
+
+![EndToEndDevDeployWorkflow](./media/image10-3.png){width="6.25in" height="3.75in"}
```
Comment (JRAlexander) [inline docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md]: The style tag is showing in the review url:  https://review.docs.microsoft.com/en-us/dotnet/standard/modern-web-apps-azure-architecture/development-process-for-azure?branch=pr-en-us-3357

**Review by JRAlexander — CHANGES_REQUESTED** [review]
A few items for review.

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+title: Architectural principles | Microsoft Docs
```
Comment (mairaw) [inline docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: nit: we're not adding | Microsoft Docs to docs

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+---
+title: Architectural principles | Microsoft Docs 
+description: Architect Modern Web Applications with ASP.NET Core and Azure | Architectural principles
```
Comment (mairaw) [inline docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: add this to the SEO work item to update these too

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+keywords: Docker, Microservices, ASP.NET, Container
```
Comment (mairaw) [inline docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: nit: we're not adding these anymore

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+author: ardalis
+ms.author: wiwagn
+ms.date: 10/06/2017
```
Comment (mairaw) [inline docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: ms.product and ms.technology: do you need those?

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+In classes, encapsulation is achieved by limiting outside access to the class's internal state. If an outside actor wants to manipulate the state of the object, it should do so through a well-defined function (or property setter), rather than having direct access to the private state of the object. Likewise, application components and applications themselves should expose well-defined interfaces for their collaborators to use, rather than allowing their state to be modified directly. This frees the application's internal design to evolve over time without worrying that doing so will break collaborators, so long as the public contracts are maintained.
+
+### Dependency Inversion
+
+The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution. This produces a direct dependency graph. That is, if module A calls a function in module B, which calls a function in module C, then at compile time A will depend on B which will depend on C, as shown in Figure 4-1.
+
+![](./media/image4-1.png)
```
Comment (mairaw) [inline docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: media files should have friend names and alt text. might be work item for later too

File: docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md (hunk 1 above)
Comment (mairaw) [reply docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md]: this image text should say end-to-end

**Review by mairaw — COMMENTED** [review]
Some quick comments re. metadata. should we create special ms.technology values for each ebook?

File: docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+Figure 11-2 shows an example reference architecture. This diagram describes a recommended architecture approach for a Sitecore content management system website optimized for marketing.
+
+![](./media/image11-2.png)
+
+**Figure 11-2.** Sitecore marketing website reference architecture.
+
+**References – Azure Hosting Recommendations**
+
+-   Azure Solution Architectures\
+    <https://azure.microsoft.com/en-us/solutions/architecture/>
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md]: done across all files.

File: docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md (hunk 1 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md]: both fixed.

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md (hunk 2 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: fixed across all files.

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md (hunk 3 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: issue updated.

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md (hunk 4 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: fixed

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md (hunk 5 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: updated

File: docs/standard/modern-web-apps-azure-architecture/architectural-principles.md (hunk 6 above)
Comment (BillWagner) [reply docs/standard/modern-web-apps-azure-architecture/architectural-principles.md]: Issue #3480 created

**Comment by BillWagner** [comment]
@JRAlexander I made all the updates you and @mairaw requested. Want to give it another look?

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**